implementations/rust/ockam/ockam_api/src/control_api/openapi.rs

```diff
@@ -24,12 +26,93 @@ impl Modify for Authentications {
+
+## Authentication
+Only a simple bearer token is required to authenticate. The token is passed in the `Authorization`
+header: `Authorization: Bearer my-secret-token`.
+
+## Development
+The easiest way to get development started is to use this API with the Ockam Command and run a single node acting both as a frontend and backend.
+```sh
+ockam node create --foreground -vv --launch-configuration '{
+  "start_default_services": true,
+  "startup_services": {
+    "control_api": {
+      "authentication_token": "my-secret-token",
+      "backend": true,
+      "frontend": true,
+      "http_bind_address": "127.0.0.1:8080"
+    }
+  }
+}'
+```
+
+When an undocumented error is returned, such as internal server error, a message contained by `ErrorResponse` is usually present for debugging purposes.
+It's highly recommended to check the logs of the Ockam nodes for more information.
+"#;
+
 #[derive(OpenApi)]
 #[openapi(
     info(
-        title = "Ockam Control API",
+        title = "Ockam Node Control API",
         version = "0.1.0",
-        description = "API to control Ockam nodes",
+        description = MAIN_DESCRIPTION,
+    ),
+    tags((
+        name = "Portals",
+        external_docs(
+            url = "https://docs.ockam.io/reference/command/advanced-routing#portal",
+            description = "Learn more about Portals on Ockam Command documentation"
+        ),
+        description =
+"
+A TCP Inlet and TCP Outlet together form a Portal, working hand in hand with Relays. A TCP Inlet
+defines where a node, running on another machine, listens for connections. The Inlet's route
+provides information on how to forward traffic to the Outlet (its address). Relays allow you to
+establish end-to-end protocols with services that operate in remote private networks.
+"
+        ),(
+        name = "Relays",
+        external_docs(
+            url = "https://docs.ockam.io/reference/command/advanced-routing#relays",
+            description = "Learn more about Relays on Ockam Command documentation"
+        ),
+        description =
+"
+Relays make it possible to establish end-to-end protocols with services operating in a remote
+private networks, without requiring a remote service to expose listening ports to an outside hostile
+network like the Internet.
+",
+        ),(
+        name = "Tickets",
+        description =
+"
+The ticket is plain text representing a one-time use token and the non-sensitive data about the
+Project, like the route to reach it and the Project Identity Identifier, which will be used to
+validate the Project Identity. The ticket itself can be stored in an environment variable, or a
+file.
+",
+        ),(
+            name = "Authority Members",
+            description =
+"
+An Ockam Authority is a Ockam node running a set of services dedicated to managing credentials
+and identities.
+The Ockam Authority keeps a list of members, which are entitled to receive credentials with proper
```

Comment:
Do we have a place where we document:

 - The predefined roles (who can create members?).
 - The predefined attributes (like `ockam-relay`).